akka-actor/src/main/scala/akka/actor/AbstractActor.scala

```diff
@@ -108,23 +108,21 @@ object AbstractActor {
  * <p/>
  * Example:
  * <pre>
- * public class MyActor extends AbstractActor {
- *   int count = 0;
- *
- *   public MyActor() {
- *     receive(receiveBuilder()
- *       .match(Double.class, d -> {
- *         sender().tell(d.isNaN() ? 0 : d, self());
- *       })
- *       .match(Integer.class, i -> {
- *         sender().tell(i * 10, self());
- *       })
- *       .match(String.class, s -> s.startsWith("foo"), s -> {
- *         sender().tell(s.toUpperCase(), self());
- *       })
- *       .build();
- *     );
- *   }
```

Comment:
another data point against having uncompiled code in scaladoc

Reply:
yeah, a little misleading.